### PR TITLE
add horror page

### PR DESCRIPTION
**Pull request by mortazavian**

### CODE REVIEW

File: src/views/genre/Horror.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+      <Movie :information="movies[index]" />
+    </div>
+  </div>
+</template>
+<script>
+import Movie from "../../components/Movie.vue";
+import axios from "axios";
+export default {
+  name: "Horro",
```
Comment (aliiimaher) [inline src/views/genre/Horror.vue]: bad writing

**Review by aliiimaher — CHANGES_REQUESTED** [review]
not horro, horror, line 14

File: src/views/genre/Horror.vue (hunk 1 above)
Comment (mortazavian) [reply src/views/genre/Horror.vue]: I was not focus. my bad

**Review by aliiimaher — COMMENTED** [review]
Daskhosh Now!

**Pull request merged**